<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2ABDV09" name="Default test" tests="1" failures="1" timestamp="11 Jul 2016 16:07:07 GMT" time="2.258" errors="0">
  <testcase name="test" time="0.261" classname="top.fzqblog.test.service.ServiceTest">
    <failure type="top.fzqblog.exception.BussinessException" message="邮箱已存在">
      <![CDATA[top.fzqblog.exception.BussinessException: 邮箱已存在
]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Default test -->
